src/components/gltf-model-plus.js

```diff
@@ -449,6 +468,38 @@ class GLTFHubsPlugin {
         }
 
         node.extras.gltfIndex = i;
+
+        if (qsTruthy("newLoader")) {
```

Comment:
Just a reminder: We will replace qstruthy newLoader with `shouldUseNewLoader()` in https://github.com/mozilla/hubs/pull/6042